junit-platform-engine/src/main/java/org/junit/platform/engine/support/store/NamespacedHierarchicalStore.java

```diff
@@ -84,10 +87,13 @@ public NamespacedHierarchicalStore<N> newChild() {
 	 * stored values in reverse insertion order.
 	 *
 	 * <p>Closing a store does not close its parent or any of its children.
+	 * <p>Closing an already closed store is a no-op.
+	 * @throws IllegalStateException if Modifying or querying from an already
+	 * closed store.
```

Comment:
```suggestion
```

This method does not throw the exception. So please remove this.

However, you need to declare the `@throws` Javadoc for all methods that _do_ throw the exception, and modify the explanation to be specific to the action of each method.